datastore-design/go-grpc-demo/storagebackend/timescaledb/findtimeseries.go

```diff
@@ -0,0 +1,127 @@
+package timescaledb
+
+import (
+	"datastore/datastore"
+	"fmt"
+
+	postgis "github.com/cridenour/go-postgis"
+	_ "github.com/lib/pq"
+)
+
+// matchAnyValCond creates the sub-condition of a WHERE clause that checks if string column 'name'
+// matches any of a set of values.
+// Returns the sub-condition to be appended to a WHERE clause.
+func matchAnyValCond(name string, values []string) string {
+	if len(values) == 0 {
+		return ""
+	}
+	inArg := ""
+	for i, value := range values {
+		if i > 0 {
+			inArg += ","
+		}
+		inArg += fmt.Sprintf("'%s'", value)
+	}
+	return fmt.Sprintf(" AND %s IN (%s)", name, inArg)
```

Comment:
SQL injection risk? I think Placeholder variables should be used for the user provided arguments?
https://go.dev/doc/database/sql-injection

Reply:
Dammit. I'm usually very conscious about [SQL injection](https://www.explainxkcd.com/wiki/index.php/327:_Exploits_of_a_Mom), but missed this one.